SharedSources/VLCPlaybackInfoSubtitlesFetcherViewController.m

```diff
@@ -142,6 +142,7 @@ - (void)themeDidChange
     [self.navigationItem.titleView setTintColor:colors.navigationbarTextColor];
     self.view.backgroundColor = self.tableView.backgroundColor = colors.background;
     [self.tableView reloadData];
+    [self setNeedsStatusBarAppearanceUpdate];
```

Comment:
I didn't notice scenario where it may affect us. It is done only for similarity.
I've checked all usages of `preferredStatusBarStyle` overrides - and added this if result depends on theme into `themeDidChange` or similar method. It was already done in many (but not all) places.